src/VisualStudio/CSharp/Impl/PackageRegistration.pkgdef

```diff
@@ -192,3 +192,10 @@
 "Package"="{13C3BBB4-F18F-4111-9F54-A0FB010D9194}"
 "SortPriority"=dword:00000064
 @="Microsoft Visual C#"
+
+// CacheTag value should be changed when registration file changes
```

Comment:
Is it autogenerated?  Or does it have to be manually updated each time we add / change / remove a setting?

Manual updating seems problematic

Reply:
It needs to be manually updated. But I think from the doc it only affects F5 deployment/Testing.

I learned from @JoshuaBStevens  that there is a way to config this to be auto-genereated (There is a VS side PR sample to do this, but it is using internal tasks)
So either, 1. Investigate how to do this publicly. 2. Later when all things are moved to gladstone this could also be auto-generated.

All of these look like another work.